### ISSUE TITLE
[Bug Report]  Archive error: Couldn't find directory end signature in the last 65KB

### ISSUE THREAD

**Issue by supasonix-zuv** [hkalexling/Mango#299]
**Describe the bug**
Some archives are unreadable with the error message `Archive error: Couldn't find directory end signature in the last 65KB`. It tends to be about once per 20 volume series. There is no thumbnail as well.

**To Reproduce**
1. Import an archive
2. It may or may not work
3. See error

**Expected behavior**
The archive loads and reads properly.

**Environment (please complete the following information):**
 - OS: Ubuntu 20.04
 - Browser: Occurs on both Firefox and Tachiyomi's extension
 - Mango Version: 0.26.1

**Docker (if you are running Mango in a Docker container)**
` mango:                                       
  image: hkalexling/mango
  container_name: mango        
  restart: unless-stopped
  expose:                                       
   - 9000                                 
  ports:                  
   - 9000:9000             
  volumes:    
   - /mnt/SSD/mango:/root/mango
   - /mnt/SSD/mango/.config:/root/.config/mango
   - /mnt/HDD/Reading/Manga:/root/mango/library
`
**Additional context**
It's also not a problem with the archives, I can literally copy and paste the archive and it works just fine. Running a checksum shows that they are exactly the same.

**Comment by hkalexling** [comment]
Can you try deleting the `/mnt/SSD/mango/library.yml.gz` file and see if it resolves the issue? This is a temporary cache for the library and will be regenerated.

**Comment by supasonix-zuv** [comment]
It didn't seem to do anything, even after restarting the docker container the issue persists

**Comment by wandermaus** [comment]
Same issue for me. Also tried to delete the `/mnt/SSD/mango/library.yml.gz` file.
Im using .cbz files and they are all working in other readers.

**Comment by Leeingnyo** [comment]
Is it different with https://github.com/hkalexling/Mango/issues/112#issuecomment-712103777?
cbz files archived with 7zip are not yet supported, aren't they?

**Comment by wandermaus** [comment]
I "re-achieved" them with 7zip using the .zip extension and rename them to .cbz. Seems to work after that.

**Comment by hkalexling** [comment]
@Leeingnyo Yeah I was suspecting that to be an 7z issue as well, but @supasonix-zuv mentioned that the exact copy of the archive worked, which is really strange. I have no idea why it's happening and can't reproduce it :thinking:

**Comment by alpharde** [comment]
I'm getting this too. I can provide a file that triggers this issue.

**Comment by hkalexling** [comment]
@alpharde Are those 7zip files? You might want to re-archive them into standard zip files. See #112

**Comment by alpharde** [comment]
Sorry, forgot about this issue. The files were indeed corrupt.
